.github/workflows/test.yml

```diff
@@ -104,31 +87,21 @@ jobs:
           echo "REQUESTS_HTTP_PROXY_AUTH_USER=test" >> $GITHUB_ENV
           echo "REQUESTS_HTTP_PROXY_AUTH_PASS=pass" >> $GITHUB_ENV
 
-      - name: Ensure the HTTPS test instance on Heroku is spun up
-        run: curl -s -I http://requests-php-tests.herokuapp.com/ > /dev/null
-
-      - name: Check mitmproxy version
-        run: mitmdump --version
-
-      - name: Ping localhost domain
-        run: ping -c1 localhost
-
-      - name: Access localhost on port 8080
-        run: curl -i http://localhost:8080
-
-      - name: Access localhost on port 9002
-        run: curl -i http://localhost:9002
-
-      - name: Show PHPUnit version
-        run: vendor/bin/phpunit --version
+      - name: Checkproxy server
+        run: |
+          mitmdump --version
+          curl -s -I http://requests-php-tests.herokuapp.com/ > /dev/null
+          ping -c1 localhost
+          curl -i http://localhost:8080
+          curl -i http://localhost:9002
```

Comment:
These separate steps make CI output very long but provide no additional info.

Reply:
> These separate steps make CI output very long but provide no additional info.

I do not agree. If anything breaks in one of those steps, it is immediately obvious in which step the failure occurred and where to start debugging.

Debugging becomes a lot harder when multiple commands are combined in one step.